### PR TITLE
release: 3.11.2

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR is to generate a new release of `code-server` at `3.11.2`.

Notes: we're publishing this because we had a hiccup releasing `3.11.1` to npm.

## Screenshot

https://user-images.githubusercontent.com/3806031/128785424-ad2397b9-5d45-499f-a3b9-cd66a712a2ff.mov

## TODOs

Follow "Publishing a release" steps in `ci/README.md`

- [ ] publish release and merge PR
- [ ] update the AUR package

### CODE REVIEW

**Review by jawnsy — APPROVED** [review]
nice

**Comment by jsjoeio** [comment]
Heads-up. I am moving back to draft because I'm seeing the `ripgrep` error. I think this is a direct result of the symlink removal. I'm not sure why it didn't show up in the PR where we removed it while testing. I think I'll check-in with @code-asher tomorrow

```javascript
 ERR spawn /Users/jp/Dev/code-server/release-packages/code-server-3.11.2-macos-amd64/lib/vscode/node_modules/vscode-ripgrep/bin/rg ENOENT: Error: spawn /Users/jp/Dev/code-server/release-packages/code-server-3.11.2-macos-amd64/lib/vscode/node_modules/vscode-ripgrep/bin/rg ENOENT
    at Object.M [as deserializeSearchError] (http://localhost:8080/static/90f633006e1edd78423a1ddc6e9651396c83edca/Users/jp/Dev/code-server/release-packages/code-server-3.11.2-macos-amd64/lib/vscode/out/vs/workbench/workbench.web.api.js:2000:5630)
    at http://localhost:8080/static/90f633006e1edd78423a1ddc6e9651396c83edca/Users/jp/Dev/code-server/release-packages/code-server-3.11.2-macos-amd64/lib/vscode/out/vs/workbench/workbench.web.api.js:2003:5041
    at async http://localhost:8080/static/90f633006e1edd78423a1ddc6e9651396c83edca/Users/jp/Dev/code-server/release-packages/code-server-3.11.2-macos-amd64/lib/vscode/out/vs/workbench/workbench.web.api.js:2003:3078
    at async li.getFileSearchResults (http://localhost:8080/static/90f633006e1edd78423a1ddc6e9651396c83edca/Users/jp/Dev/code-server/release-packages/code-server-3.11.2-macos-amd64/lib/vscode/out/vs/workbench/workbench.web.api.js:2478:11248)
    at async Promise.all (index 0)
    at async li.doFileSearch (http://localhost:8080/static/90f633006e1edd78423a1ddc6e9651396c83edca/Users/jp/Dev/code-server/release-packages/code-server-3.11.2-macos-amd64/lib/vscode/out/vs/workbench/workbench.web.api.js:2478:10892)
```

File: docs/README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ When done, the install script prints out instructions for running and starting
 code-server.
 
 We also have an in-depth [setup and
-configuration](https://coder.com/docs/code-server/v3.11.1/guide) guide.
+configuration](https://coder.com/docs/code-server/v3.11.2/guide) guide.
 
 ## TLS and authentication (beta)
```
Comment (galesky) [inline docs/README.md:56]: This URL will break as per https://github.com/cdr/code-server/pull/3953

Comment (galesky) [reply docs/README.md:56]: Can't really comment on the correct line - I mean L58

Comment (jsjoeio) [reply docs/README.md:56]: Going to do in a follow-up PR since I'm closing this: https://github.com/cdr/code-server/issues/3963

**Pull request closed without merging**